android/java/proguard.flags

```diff
@@ -1,3 +1,5 @@
+-keep class com.google.android.datatransport.runtime.scheduling.** { *; }
```

Comment:
We needed to add proguard rule because of https://source.chromium.org/chromium/chromium/src/+/main:third_party/angle/third_party/android_deps/buildSrc/src/main/groovy/ChromiumDepGraph.groovy;l=39. and for some reason it seems to be affecting only android 8.